_common/setupMS.js

```diff
@@ -33,5 +33,7 @@ function setupMS(params) {
 //  global.config.apiToken = process.env.SHIPPABLE_API_TOKEN;
 
   global.config.apiUrl = 'https://alphaapi.shippable.com';
-  global.config.apiToken = 'b799011c-f9ed-4a4a-9231-041132d45d5e';
+  global.config.apiToken = 'fcc04768-8a11-40bc-9b68-b8ff385b064c';
```

Comment:
this is bad. Please delete the token. I did not remember that i had this. I will delete my tokens